spec/presenters/school_vacancies_presenter_spec.rb

```diff
@@ -35,7 +35,8 @@
 
     it 'returns sorted draft vacancies' do
       sort.update(column: 'job_title', order: 'desc')
-      expect(presenter.vacancies).to eq(draft_vacancies.sort_by { |vacancy| vacancy['job_title'] }.reverse)
+      vacancy_ids = draft_vacancies.sort_by { |vacancy| vacancy['job_title'] }.reverse.pluck(:id)
+      expect(presenter.vacancies.pluck(:id)).to eq(vacancy_ids)
```

Comment:
Think we could get these pulled out into their own PR?

Reply:
Yep, we don't need this now 👍